package/InternalEAWGen.roc

```diff
@@ -0,0 +1,140 @@
+## The purpose of this file is to generate the InternalEAW.roc file.
+##
+## This file will read the test data from `data/EastAsianWidth-15.1.0.txt`
+## parse it and then generate function to test the East Asian Width property of a code point.
+app [main] {
+    pf: platform "https://github.com/roc-lang/basic-cli/releases/download/0.11.0/SY4WWMhWQ9NvQgvIthcv15AUeA7rAIJHAHgiaSHGhdY.tar.br",
+}
+
+import pf.File
+import pf.Task exposing [Task]
+import pf.Arg
+import "data/EastAsianWidth-15.1.0.txt" as file : Str
+import Helpers
+
+EawRange : (Str, Str, Str)
```

Comment:
Added an alias for this so it is easier to follow what the below functions are doing and add a type annotation. In general I think most of the top-level declarations should be annotated.